src/main/java/org/kuali/kfs/module/purap/document/PurchaseOrderDocument.java

```diff
@@ -1022,9 +1047,6 @@ public void setRecurringPaymentFrequency(RecurringPaymentFrequency recurringPaym
     }
 
     public PaymentTermType getVendorPaymentTerms() {
-        if (ObjectUtils.isNull(vendorPaymentTerms) || ObjectUtils.isNull(vendorPaymentTerms.getVendorPaymentTermsCode())) {
-            this.refreshReferenceObject("vendorPaymentTerms");
-        }
```

Comment:
This appears to be a customization that we had added back in 2011 to fix KITI-2775. Should it be kept in, or do you think we can try to remove it now and then have the functionals validate it accordingly?

Reply:
Good question. Could you please create a user story for this? Thank you.

Reply:
I've created KFSPTS-16911 to perform further analysis into whether this customization is still needed. In the meantime, I'm fine with this PR removing it.